src/hyperlight_host/src/mem/shared_mem.rs

```diff
@@ -122,7 +122,7 @@ unsafe impl Send for ExclusiveSharedMemory {}
 /// unit that likely can't be discovered by the compiler) that _rust_
 /// users do not perform racy accesses to the guest communication
 /// buffers that are also accessed by HostSharedMemory.
-#[derive(Debug)]
```

Comment:
I believe the GuestSharedMemory was purposefully not Clone because it was intended to reflect the unique ownership at any time of the guest side access to the shared memory. This may actually be somewhat important for soundness, actually; I can't recall.

Reply:
Removed ✅ . Thanks Lucy for the help again!